### ISSUE TITLE
Quality of encoding as percent from original size

### ISSUE THREAD

**Issue by dohn** [RandomEngy/VidCoder#1086]
### Feature details

I would like to ask for a feature that I have never seen in any encoding software.
Let's say I have 50 different video files. They have different file sizes, varying from let's say 100 MB to 2 GB.
Now, I would like to BATCH convert them in such way that each target file has 50% file size from the original one.
In other words, I would like that under Video Encoding tab, to add another radio button for Quality. Right under Target size (MB), add a radio button titled Target size from original (%) with an input box for percent.
That would make my day.
Thank you.

**Comment by RandomEngy** [comment]
I generally encourage people to take advantage of the quality targeting option; that does a very good job of handling diverse inputs and getting an acceptable quality output with minimal file size. What advantage would this feature have over quality targeting? If you just blindly reduce the bitrate on all the videos you might end up with some with poor quality, if they started out with a low bitrate to begin with.

**Comment by dohn** [comment]
Your assumption came out of the blue. I do not "blindly" reduce the bitrate on all videos. I know exactly what initial bitrate the videos have.

**Comment by RandomEngy** [comment]
Sorry about that. I still think that setting a quality target will give you a better size/quality tradeoff. Videos with lower complexity can be compressed to a smaller size than those with lots of motion and high visual complexity.

**Comment by dohn** [comment]
I could ask myself the same thing about the utility of a FIXED target file size when converting a big number of videos. Some might be smaller, some might be bigger. Remember, I am talking about BATCH conversion. Only a percent value of the original size makes sense. Let us worry about the possible poor quality. Everybody knows that quality vs file size is a matter of trade off. Except that I miss that possibility as a separate target quality criteria.

**Comment by RandomEngy** [comment]
Yes, your proposal would be better than trying to use target size. But I think quality targeting would do an even better job in your case.
